cypress/integration/adminTasks.js

```diff
@@ -0,0 +1,36 @@
+import TaskPage from '../helpers/TaskPage'
+
+// T-3
+describe('Task creation', () => {
+
+    it('admin should be able to add new tasks', () => {
+        cy.loginAs('Admin')
+        TaskPage.visit()
+        TaskPage.getCreateButton().click()
+
+        cy.url().should('include', '/tasks/create');
+
+        
+        const randomName = Math.random().toString(36).substring(9);
+        TaskPage.getTaskNameInput().type(randomName)
+        TaskPage.getDescriptionInput().type(randomName)
+
+        const billToClient = TaskPage.getBillToClient()
+        billToClient.should('have.length', 1)
+        billToClient.click()
+        TaskPage.getBillToClientOption('Yes').click()
+
+        TaskPage.getHourlyRate().type(Math.random()*15)
```

Comment:
I would add spaces between * signs and other mathematical operators because it's easier to read code that way